Generator/Logic/SSettingsEnums.cs

```diff
@@ -83,4 +83,12 @@ public enum GoronMinesEntrance
         NoWrestling = 1,
         Open = 2,
     }
+     public enum ItemScarcity
+    {
+        Null = 0,
```

Comment:
What is the purpose of Null? It seems like Vanilla could be 0. Vanilla is identical to the behavior before this dropdown was added.